features/verblijfplaats.feature

```diff
@@ -0,0 +1,140 @@
+      | datumInschrijvingInGemeente.datum    | 2002-07-01             |
+      | functieAdres.code                    | B                      |
+      | straat                               | Spui                   |
+      | korteNaam                            | Spui                   |
+      | huisnummer                           | 70                     |
+      | postcode                             | 2511BT                 |
+      | woonplaats                           | 's-Gravenhage          |
+      | adresregel1                          | Spui 70                |
+      | adresregel2                          | 2511 BT  'S-GRAVENHAGE |
+      | datumVan.type                        | Datum                  |
+      | datumVan.datum                       | 2022-01-28             |
+
+  Rule: Een verblijfplaats is een binnenlandse locatie wanneer locatiebeschrijving (12.10) een waarde heeft
+    - veld 'type' krijgt de waarde 'Locatie' 
+
+    Abstract Scenario: verblijfplaats binnenlandse locatie
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 555550001 |
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      | naam                                | waarde                     |
+      | gemeenteVanInschrijving (09.10)     | 0546                       |
+      | datumInschrijvingInGemeente (09.20) | 20020701                   |
+      | functieAdres (10.10)                | W                          |
+      | datumAanvangAdreshouding (10.30)    | 20220128                   |
+      | locatiebeschrijving (12.10)         | Woonboot in de Grote Sloot |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                             |
+      | type                | RaadpleegMetBurgerservicenummer    |
+      | burgerservicenummer | 555550001                          |
+      | fields              | burgerservicenummer,verblijfplaats |
+      Dan heeft de persoon met burgerservicenummer '555550001' de volgende 'verblijfplaats' gegevens
+      | naam                              | waarde                     |
+      | type                              | Locatie                    |
+      | gemeenteVanInschrijving.code      | 0546                       |
+      | datumInschrijvingInGemeente.type  | Datum                      |
+      | datumInschrijvingInGemeente.datum | 2002-07-01                 |
+      | functieAdres.code                 | W                          |
+      | locatiebeschrijving               | Woonboot in de Grote Sloot |
+      | woonplaats                        | Leiden                     |
+      | adresregel1                       | Woonboot in de Grote Sloot |
+      | adresregel2                       | LEIDEN                     |
+      | datumVan                          | 2022-01-28                 |
```

Comment:
datumVan komt niet voor in de OAS specificatie van Locatie? Wat is niet correct?

Reply:
datumVan moet ook voorkomen bij een Locatie. heb issue hiervoor gemaakt: #976